python/golem_task_api/server.py

```diff
@@ -156,7 +156,12 @@ def __init__(
     @forward_exceptions()
     async def RunBenchmark(self, stream):
         await stream.recv_message()
-        score = await self._handler.run_benchmark(self._work_dir)
+        try:
+            score = await self._handler.run_benchmark(self._work_dir)
+        except Exception as e:
```

Comment:
Isn't `@forward_exception` already doing this job?

Reply:
i had some issues with showing the errors so i added this block

we can still use it for setting status or executing finally ( see TODO next line ) or remove it and let `@forward_exception` handle it

Reply:
So, what do we do with this?

Reply:
for now i'll remove it, we can always add it back when we want to set a status

Reply:
OK, fine.